src/main/java/course_project/Main.java

```diff
@@ -0,0 +1,7 @@
+package course_project;
+
+public class Main {
```

Comment:
Quite a strange ship here
![image](https://user-images.githubusercontent.com/58872509/134312539-416aac50-438c-4a74-95f5-1581f6fca4cf.png)

Reply:
Good job! The game is playable and has a nice description, the code is pretty readable. It seems there are some bugs, good that you have automatic ship placement, the interface is cool. You show knowledge of Java Core and Java libraries. Abstractions and architecture is TBD

To make it better: add some features (like automatically mark cells around destroyed ship as miss), introduce the architecture, and more abstractions.

Ping me when you resolved this bug and tested your app manually for other bugs (like, what happens if the computer hits a cell already marked as a miss?)

Reply:
The game shows me the opponent's field in the beginning? Have you played your game yourself?) 
![image](https://user-images.githubusercontent.com/58872509/134488832-6f039b7b-2445-4517-b9df-90c937d036e9.png)

Reply:
Opt: Also, there are no messages what happened - did computer (or me) hit or miss, how many times computer shot and etc...

Reply:
Approved!